pkg/cli/config/config.go

```diff
@@ -196,9 +202,9 @@ func (c *Config) ServerVersion() (*Version, error) {
 		return nil, err
 	}
 
-	resp, err := c.DoRequest(req)
+	resp, err := c.DoRequestWithTimeout(req, fastTimeout)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("access %s failed", c.Server)
```

Comment:
Do we need to print original error

Reply:
No need, while use --debug flag, will log the response, and can check the failed reason by response body.